modules/notification/ui/ui.go

```diff
@@ -105,6 +105,17 @@ func (ns *notificationService) NotifyPullRequestReview(pr *models.PullRequest, r
 	_ = ns.issueQueue.Push(opts)
 }
 
+func (ns *notificationService) NotifyPullRequestPushCommits(doer *models.User, pr *models.PullRequest, comment *models.Comment) {
+	var opts = issueNotificationOpts{
+		IssueID:              pr.IssueID,
+		NotificationAuthorID: doer.ID,
+	}
+	if comment != nil {
+		opts.CommentID = comment.ID
```

Comment:
Shouldn't `comment` be always present for this notification?

Reply:
Hmm, Just protect bug ..., If it's null, how to get id? I will check whether it's necessary again ...